.ci/azure-pipelines/build/windows.yaml

```diff
@@ -8,18 +8,13 @@ steps:
   - script: |
       echo ##vso[task.prependpath]%BOOST_ROOT_1_69_0%\lib
     displayName: 'Include Boost Libraries In System PATH'
-  - script: |
-      set
-    displayName: 'Print Environment Variables'
+  - pwsh: Get-PSDrive
+    displayName: "Check free space"
   - script: |
       vcpkg.exe install eigen3 flann gtest qhull --triplet %PLATFORM%-windows && vcpkg.exe list
     displayName: 'Install C++ Dependencies Via Vcpkg'
   - script: |
-      rmdir %VCPKG_ROOT%\downloads /S /Q
-      rmdir %VCPKG_ROOT%\packages /S /Q
-    displayName: 'Free Up Space'
-  - script: |
-      mkdir %BUILD_DIR% && cd %BUILD_DIR%
+      c: && mkdir %BUILD_DIR% && cd %BUILD_DIR%
```

Comment:
Since BUILD_DIR contains `c:` as prefix, is the initial `c: &&`  needed?

Reply:
Yes, the first c: change the drive. As it starts on d: drive, it wont navigate correctly if not switched to c: drive.

Reply:
I have changed to use the workingDirectory set as 'C:' which also works.